charts/hawkbit/templates/deployment.yaml

```diff
@@ -37,7 +37,7 @@ spec:
             - name: SPRING_PROFILES_ACTIVE
               value: "{{ .Values.spring.profiles }}"
             - name: "SPRING_DATASOURCE_URL"
-              value: "jdbc:mysql://{{ if .Values.mysql.enabled }}{{ .Release.Name }}-mysql{{ else }}{{ .Values.env.springDatasourceHost }}{{ end }}:3306/{{ .Values.env.springDatasourceDb }}"
+              value: "{{ if .Values.env.springDatasourceUrl }}{{ .Values.env.springDatasourceUrl }}{{ else }}jdbc:mysql://{{ if .Values.mysql.enabled }}{{ .Release.Name }}-mysql{{ else }}{{ .Values.env.springDatasourceHost }}{{ end }}:3306/{{ .Values.env.springDatasourceDb }} {{end}}"
```

Comment:
An extra space character got added here in the last part of the string:
`[...].springDatasourceDb }} {{end}}"`.

In general, this long line with the 2 `if` conditions is quite hard to read.
How about making multiple lines out of it?
E.g.
```
              {{- if .Values.env.springDatasourceUrl }}
              value: "{{ .Values.env.springDatasourceUrl }}"
              {{- else }}
              value: "jdbc:mysql://{{ if .Values.mysql.enabled }}{{ .Release.Name }}-mysql{{ else }}{{ .Values.env.springDatasourceHost }}{{ end }}:3306/{{ .Values.env.springDatasourceDb }}"
              {{- end}}
```

Reply:
done